common/tools/dev-tool/src/templates/sampleReadme.md.ts

```diff
@@ -6,7 +6,6 @@ import YAML from "yaml";
 
 import prettier from "prettier";
 
-import { MIN_SUPPORTED_NODE_VERSION } from "../util/sampleConfiguration";
```

Comment:
if `MIN_SUPPORTED_NODE_VERSION` is not being used elsewhere, we probably should delete it cc @witemple-msft

Reply:
It is being used to set the `engines` field in the generated package.json files for samples. So, I kept it.